src/infra/main/template.yaml

```diff
@@ -197,6 +215,258 @@ Resources:
       AliasName: !Sub "alias/${AWS::StackName}-SQSEncryptionKey"
       TargetKeyId: !GetAtt SQSEncryptionKey.Arn
 
+  AccountDeletionProcessorFunction:
+    Type: AWS::Serverless::Function
+    # checkov:skip=CKV_AWS_116: DLQ is not appropriate for a Lambda invoked by an API
+    # checkov:skip=CKV_AWS_115: Function is not configured for function-level concurrent execution Limit
+    Properties:
+      FunctionName: !Sub '${AWS::StackName}-AccountDeletionProcessorFunction'
+      Description: "A lambda function that updates an account status indicating that the account has been deleted"
+      CodeUri: ../../../dist/handlers/account-deletion-processor-handler
+      Timeout: 10
```

Comment:
Is there a point in having the visibility timeout value on the queue as high as 60, but have the lambda timeout at 10?

Reply:
Nope agreed that shouldn't work? Has this been tested @jessica-li-01

Reply:
Changed- thanks! :)